tests/test_database_sqlite.py

```diff
@@ -1,115 +1,116 @@
-"""Tests for the DatabaseSqlite class."""
-import asyncio
-
-import unittest
-import asynctest
+"""Tests for DatabaseSqlite"""
+import pytest
 import asynctest.mock as amock
 
+import asyncio
+
 from opsdroid.database.sqlite import DatabaseSqlite
 from opsdroid.cli.start import configure_lang
 
-
-class TestDatabaseSqlite(unittest.TestCase):
-    """A database test class.
+"""A database test class.
 
```

Comment:
Let's remove these comments. My reasoning behind that is that we already have the comment on line 1 saying what the test is about 😄